examples/cosmos-etcd/readme.md

```diff
@@ -1,9 +1,63 @@
 # Kubernetes on Cosmos
 
-This example deployment allows you to bootstrap a kubernetes cluster on Azure that uses Cosmos instead of locally installed and configured etcd.
+The Azure Kubernetes Engine (aks-engine) generates [ARM](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-overview) (Azure Resource Manager) templates for Kubernetes clusters on Microsoft Azure. The input to aks-engine is a cluster definition file which describes the desired cluster, including orchestrator, features, and agents. 
```

Comment:
I am not sure if this paragraph is necessary as a user exploring these examples would already know what aks-engine does.

Reply:
Removed.